app/Models/Author.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Author extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    public function authorBooks()
```

Comment:
It should be 'books', rather than 'authorBooks', because the Model is already representing 'Author'.